src/website/apps/faqs/models.py

```
@@ -4,8 +4,20 @@
 
 
 class FAQ(models.Model):
+    CATEGORY_CHOICES = [
+        ('general', 'General'),
+        ('hardware', 'Hardware'),
+        ('software', 'Software'),
+    ]
```

Ruff (0.14.7)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)